app/models/Cookbooks.py

```diff
@@ -0,0 +1,20 @@
+import uuid
+from sqlmodel import Field, Relationship, SQLModel
+
+
+
+# Cookbook Table
+class Cookbook(SQLModel, table=True):
+    id: int = Field(default=None, primary_key=True)
+    owner_id: int = Field(foreign_key="user.id", unique=True)
+    cover: str | None = Field(default=None)
+    thumbnail: str | None = Field(default=None)
+    title: str | None = Field(default=None)
+    description: str | None = Field(default=None)
+    # 4th uuid generator
```

Comment:
The two comments here don't seem the most helpful to me personally.

Reply:
is there a specific comment format we need for the models to explain what they are? I think first comment makes sense but could probs be elaborated on for more details like to show each of the fields. Albeit this is a hackathon so no need to really do all that for now